### DISCUSSION TITLE
Shell.PresentationMode="NotAnimated" not working on IOS

### DISCUSSION THREAD

**Discussion by FM1973** [dotnet/maui#14765] [Q&A]
Hello.
I´m trying to disable all page transition animations.

When navigating using GoToAsync and setting animated false, this works on Android everywhere.

On IOS this only works for pages not set as a ShellContent page in appshell.xaml.
I tried to set Shell.PresentationMode="NotAnimated" but this doesn´t help on IOS.
Does anyone know how to disable the fading animation on IOS?

**Comment by k-howell** [comment]
@FM1973 Did you ever find a solution to this problem?
